### ISSUE TITLE
Why require video tags to have a track element with type=description? 

### ISSUE THREAD

**Issue by DavidMacDonald** [dequelabs/axe-core#1712]
Expectation: Should not flag video tags that are missing ```<track kind="description"...>```
Actual: Flags them as missing audio description

Motivation: There is no evidence that I've found that a kind=description attribute can be used by any existing tool to provide audio description

**Comment by WilcoFiers** [comment]
Hey David, I think you've got a point. I only know of a single media player that supports kind=description, and it's not one of the common ones. We are working on a plan on how we might be able to deprecate rules. We'll take this into consideration too.

**Comment by jeankaplansky** [comment]
Deprecate rule in 3.4 rule help.

**Comment by chandana7393** [comment]
Tested, the issue is still appearing.
![video_type_description](https://user-images.githubusercontent.com/45285854/67281583-5ca11d80-f4ed-11e9-939b-f8628dca996a.PNG)

**Tested Environment:**
Axe-coconut - 4.0.0.29504v
Chrome -  77.0.3865.120v
OS - Windows 10 64 bit.
